.github/workflows/onpush-checks.yml

```diff
@@ -24,7 +40,12 @@ jobs:
     - uses: actions/checkout@v1
       with:
         submodules: recursive
-    - name: Execute
+    - name: Cache pip
+      uses: actions/cache@v2
```

Comment:
Is this PR supposed to be on top of #326 ? (I can see that you are using the `cache` action here.)

Reply:
No, this is independent of that PR.